pom.xml

```diff
@@ -230,8 +230,8 @@
                     <validationSets>
                         <validationSet>
                             <includes>
-                                <include>**/*.windup.xml</include>
-                                <include>**/*.windup.test.xml</include>
+                                <include>**/*.awindup.xml</include>
+                                <include>**/*.awindup.test.xml</include>
```

Comment:
what's is the "a" letter in wildcard inclusion?

Reply:
A hack since I had not written the schema yet. I'll work on that now. :)

Reply:
ok, :)

Reply:
Fixed